packages/prisma/migrations/20220409195425_index_event_types_team_id_slug/migration.sql

```diff
@@ -0,0 +1,14 @@
+/*
+  Warnings:
+
+  - A unique constraint covering the columns `[teamId,slug]` on the table `EventType` will be added. If there are existing duplicate values, this will fail.
+
+*/
+-- CreateIndex
+CREATE UNIQUE INDEX "EventType_teamId_slug_key" ON "EventType"("teamId", "slug");
+
+-- RenameIndex
+ALTER INDEX "VerificationRequest.identifier_token_unique" RENAME TO "VerificationRequest_identifier_token_key";
+
+-- RenameIndex
+ALTER INDEX "VerificationRequest.token_unique" RENAME TO "VerificationRequest_token_key";
```

Comment:
These renames should be handled in another PR since these have noting to do with this change:
```suggestion
```